src/CMakeLists.txt

```diff
@@ -398,9 +398,10 @@ set(LIB_INSTALL_DIR lib${LIB_SUFFIX}) # determine correct lib folder
 
 # Install primary library
 INSTALL(  TARGETS openshot
-		ARCHIVE DESTINATION ${LIB_INSTALL_DIR}
-		LIBRARY DESTINATION ${LIB_INSTALL_DIR}
-		COMPONENT library )
+  RUNTIME DESTINATION ${LIB_INSTALL_DIR}
```

Comment:
Why put executables in the library directory?

Reply:
That's actually present in our _current_ build configs too, for Historical Reasons™.

(CMake on Windows writes DLL files to the `RUNTIME DESTINATION`, which is _meant_ to be the same directory as executables — but our packaging scripts expect them to be in the `${LIB_INSTALL_DIR}`. Because once upon a time, there was only one `DESTINATION` parameter, which had the same effect of lumping everything into the same place. ...I still hope to fix that one day, and have the Windows build put the DLL files in the `bin` directory where — as counterintuitive as it seems to those of us used to Unix norms — they actually do belong.)